oscarapi/tests/testcheckout.py

```diff
@@ -176,16 +179,56 @@ def test_checkout_header(self):
         "Prove that the user 'nobody' can checkout his cart when authenticating with header session"
         self.fail('Please add implementation')
 
-    @unittest.skip
+
     def test_anonymous_checkout(self):
-        "Can anonymous users check out a cart? If so prove it with a test"
-        self.fail('Please add implementation')
+        "Test if an order can be placed as an anonymous user."
+        response = self.get('api-basket')
+        self.assertTrue(response.status_code, 200)
+        basket = json.loads(response.content)
+        basket_url = basket.get('url')
+        basket_id = basket.get('id')
+
+        request = {
+            'basket': basket_url,
+            'guest_email': 'henk@example.com',
+            'total': '50.0',
+            'shipping_method_code': "no-shipping-required",
+            'shipping_charge': {
+                'currency': 'EUR',
+                'excl_tax':'0.00',
+                'tax':'0.00'
+            },
+            "shipping_address": {
+                "country": "http://127.0.0.1:8000/api/countries/NL/",
+                "first_name": "Henk",
+                "last_name": "Van den Heuvel",
+                "line1": "Roemerlaan 44",
+                "line2": "",
+                "line3": "",
+                "line4": "Kroekingen",
+                "notes": "Niet STUK MAKEN OK!!!!",
+                "phone_number": "+31 26 370 4887",
+                "postcode": "7777KK",
+                "state": "Gerendrecht",
+                "title": "Mr"
+            }
+        }
+
+        with mock.patch('django.conf.settings.OSCAR_ALLOW_ANON_CHECKOUT', True, create=True):
```

Comment:
+1 on that. Depending on `mock>=1.3.0` just for this already build-in functionality is not really needed.